tardis/montecarlo/montecarlo_numba/formal_integral.py

```diff
@@ -247,7 +247,12 @@ def formal_integral(
 
 class FormalIntegrator(object):
     """
-    Class containing the formal integrator
+    Class containing the formal integrator. If there is a 
```

Comment:
Look at PEP 257 for Docstring format

Reply:
Should I just have the blank line after the first sentence?

Reply:
What is model, plasma, runner, points. docstring describe init